util/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "me.champeau.jmh"
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'gRPC: Util'
+
+dependencies {
+    api project(':grpc-core')
+
+    implementation libraries.gson,
```

Comment:
Why depend on gson?

Reply:
Without it the build of grpc-services fails:

```
* What went wrong:
Execution failed for task ':grpc-services:checkUpperBoundDeps'.
> Maven version skew: com.google.code.gson:gson (2.8.9 != 2.10.1) Bad version dependency path: [project ':grpc-services', com.google.protobuf:protobuf-java-util:3.22.3] Run './gradlew :grpc-services:dependencies --configuration runtimeClasspath' to diagnose
```

Reply:
And then running `./gradlew :grpc-services:dependencies --configuration runtimeClasspath` (as suggested) gives me:

```
runtimeClasspath - Runtime classpath of source set 'main'.
...
+--- project :grpc-util
|    +--- project :grpc-core
|    |    +--- project :grpc-api (*)
|    |    +--- com.google.code.gson:gson:2.10.1 ===> v2.10.1 referenced
|    |    +--- com.google.android:annotations:4.1.1.4
|    |    +--- org.codehaus.mojo:animal-sniffer-annotations:1.23
|    |    +--- com.google.errorprone:error_prone_annotations:2.18.0
|    |    +--- com.google.guava:guava:32.0.1-android -> 32.0.1-jre (*)
|    |    +--- io.perfmark:perfmark-api:0.26.0
|    |    +--- project :grpc-context
|    |    |    \--- project :grpc-api (*)
|    |    \--- project :grpc-util (*)
|    +--- org.codehaus.mojo:animal-sniffer-annotations:1.23
|    \--- com.google.guava:guava:32.0.1-android -> 32.0.1-jre (*)
+--- com.google.protobuf:protobuf-java-util:3.22.3
|    +--- com.google.protobuf:protobuf-java:3.22.3
|    +--- com.google.code.findbugs:jsr305:3.0.2
|    +--- com.google.code.gson:gson:2.8.9 -> 2.10.1 ===> v2.8.9 referenced 
|    +--- com.google.errorprone:error_prone_annotations:2.11.0 -> 2.18.0
|    +--- com.google.guava:guava:31.1-jre -> 32.0.1-jre (*)
|    \--- com.google.j2objc:j2objc-annotations:1.3 -> 2.8
...
```

I can also fix it by making `libraries.gson` a runtimeOnly dependency of `grpc-util` and then add a comment describing why it is required?

Reply:
> I can also fix it by making libraries.gson a runtimeOnly dependency of grpc-util and then add a comment describing why it is required?

Yes. Do that for forced upgrading to fix version skew.

Reply:
Already done with a comment